dpp4/core/basyx_adapter.py

```
@@ -1,616 +1,702 @@
+                description=model.LangStringSet([
+                    model.LangString(
+                        language="en", 
+                        text=aas_data.get("description", "")
+                    )
+                ])
+            )
+            
+            # Add asset reference
+            if "asset" in aas_data:
+                asset_id = aas_data["asset"].get("id", "")
+                asset = model.Asset(
```

Ruff (0.8.2)

Local variable `asset` is assigned to but never used

Remove assignment to unused variable `asset`

(F841)